react-ui/src/components/Header.scss

```diff
@@ -5,7 +5,7 @@
   }
   
   .App-header-blue {
-    background-color: #eae8ff;
+    background-color: #b8b1fd;
```

Comment:
Changed header colour to match new colour theme decided by team.

Reply:
This is appreciated Seher! I remember this point of feedback during the previous Demo